.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
do we want to check in our `.vscode`? generally I don't think we should check in editor settings.

Reply:
I'm checking _just_ the 'you should use these extensions" suggestion and a shared workspace dictionary for spellcheck in.

I agree checking in .vscode settings that changed fonts or themes or other whatnot would be too far. This just pops up a "the authors of this repo suggest these extensions, do you want to install them?" modal. I think that's pretty useful without being intrusive. 

Especially because CI is going to run linting and fail commits that don't pass, getting that all the way into the editor is useful, and doing it via README is fine, but not the best UX.